server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/pipelines_controller.rb

```diff
@@ -18,9 +18,10 @@ module ApiV1
   module Admin
     class PipelinesController < ApiV1::BaseController
       before_action :check_admin_user_and_401
-      before_action :load_pipeline, only: [:show]
+      before_action :load_pipeline, only: [:show, :destroy]
```

Comment:
Then, isn't it better to remove the getPipelineConfig(pipelineName) from deletePipelineConfig() in PipelineConfigService and directly pass the pipeline config object so that you don't do this twice?